App Crash

```
POST /api/v1/users 400 5.366 ms - 62 
Sep 21 23:11:52 birds-staging-pr-93 app/web.1:  { UserExistsError: A user with the given username is already registered 
Sep 21 23:11:52 birds-staging-pr-93 app/web.1:      at /app/node_modules/passport-local-mongoose/index.js:244:19 
Sep 21 23:11:52 birds-staging-pr-93 app/web.1:      at /app/node_modules/mongoose/lib/query.js:2257:18 
Sep 21 23:11:52 birds-staging-pr-93 app/web.1:      at process._tickCallback (internal/process/next_tick.js:103:7) 
Sep 21 23:11:52 birds-staging-pr-93 app/web.1:    name: 'UserExistsError', 
Sep 21 23:11:52 birds-staging-pr-93 app/web.1:    message: 'A user with the given username is already registered' }  
Sep 21 23:11:52 birds-staging-pr-93 app/web.1:  events.js:160 
Sep 21 23:11:52 birds-staging-pr-93 app/web.1:        throw er; // Unhandled 'error' event 
Sep 21 23:11:52 birds-staging-pr-93 app/web.1:        ^ 
Sep 21 23:11:52 birds-staging-pr-93 app/web.1:  Error: Can't set headers after they are sent. 
Sep 21 23:11:52 birds-staging-pr-93 app/web.1:      at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:346:11) 
Sep 21 23:11:52 birds-staging-pr-93 app/web.1:      at ServerResponse.header (/app/node_modules/express/lib/response.js:719:10) 
Sep 21 23:11:52 birds-staging-pr-93 app/web.1:      at ServerResponse.send (/app/node_modules/express/lib/response.js:164:12) 
Sep 21 23:11:52 birds-staging-pr-93 app/web.1:      at ServerResponse.json (/app/node_modules/express/lib/response.js:250:15) 
Sep 21 23:11:52 birds-staging-pr-93 app/web.1:      at ServerResponse.send (/app/node_modules/express/lib/response.js:152:21) 
Sep 21 23:11:52 birds-staging-pr-93 app/web.1:      at Object.error (/app/birds/api/v1/util.js:22:46) 
Sep 21 23:11:52 birds-staging-pr-93 app/web.1:      at User.register (/app/birds/api/v1/users.js:29:25) 
Sep 21 23:11:52 birds-staging-pr-93 app/web.1:      at /app/node_modules/passport-local-mongoose/index.js:244:16 
Sep 21 23:11:52 birds-staging-pr-93 app/web.1:      at /app/node_modules/mongoose/lib/query.js:2257:18 
Sep 21 23:11:52 birds-staging-pr-93 app/web.1:      at process._tickCallback (internal/process/next_tick.js:103:7) 
Sep 21 23:11:52 birds-staging-pr-93 app/web.1:  npm ERR! Linux 3.13.0-85-generic 
Sep 21 23:11:52 birds-staging-pr-93 app/web.1:  npm ERR! argv "/app/.heroku/node/bin/node" 
```

--- a/birds/api/v1/util.js
+++ b/birds/api/v1/util.js
@@ -19,10 +19,14 @@ const message = (res, message) => {
 const error = (res, message, status) => {
     if (!status)
         console.error(message);
+    if (res.headersSent)
+        return console.error("Headers already sent on error. Ignoring response");
     return res.status(status ? status : 500).send({ error: { message: message ? message : "Unknown error." } });
 };
 
 const data = (res, data) => {
+    if (res.headersSent)
+        return console.error("Headers already sent on data. Ignoring response");
     return res.status(200).send({ data: data });
 };
 

